src/main/java/org/opentripplanner/model/StopTime.java

```diff
@@ -13,6 +13,9 @@
 
     public static final int MISSING_VALUE = -999;
 
+    /*Used to verify, whether the int value representing eg. arrival time can be converted to LocalTime object*/
+    public static final int MAX_STOP_TIME_VALUE = 86399;
```

Comment:
Ta linia nie wygląda dobrze. Nie wiem skąd się wzieła akurat taka wartość. Jeśli już musi być tu z jakiś powodów placeholder to czemu nie MAX_INT ale MISSING_VALUE?

Reply:
Też nie za bardzo podoba mi się to rozwiązanie, ale to nie placeholder - to obejście faktu, że w GTFS'ach dla nocnych linii pojawiają się godziny typu "25:30" i kiedy próbuję przekonwertować sekundy od północy wczytane z GTFS na Javowy obiekt, dostaję błąd z informacją, że przekroczona została maksymalna wartość (właśnie ta)...

Reply:
Mogę sprawdzić jeszcze, czy OTP odpytany o trasę zwraca takie dziwne godziny i jeśli nie, to czy ma jakiś wbudowany mechanizm konwertowania tego, ale obawiam się, że może nie mieć, bo te godziny typu 24:10 są zgodne ze specyfikacją...

Reply:
Problem jest faktycznie dość nietypowy, ale nie wiem, czy będzie jakaś lepsza metoda, niż taka zmienna. Ja bym ją tylko inaczej inicjalizował:
1. `SECONDS_PER_DAY - 1` z klasy `org.joda.time.DateTimeConstants` (moim zdaniem najlepsza metoda)
2. `Duration.ofDays(1).getSeconds() - 1` z klasy `java.time.Duration`
3. `24 * 60 * 60 - 1` w ostateczności.

Reply:
Zastąpiłam tą customową stałą mechanizmem wbudowanym w Javę: if (serviceTimeToWrite > LocalTime.MAX.toSecondOfDay()) - czy tak może zostać?

Reply:
super 👍